docs/05-PipelineLaunchOptions.md

```diff
@@ -158,13 +158,13 @@ The BIDS dataset to import has to:
   ```
 
 
-In cases where the `site`, `project` or `subproject` are missing from the `participant.tsv` file, `NULL` will be populated for those fields in the candidate and session tables. In those cases, later manual addition is required in order to display the entry on the frontend.
+In cases where the `site`, `project` or `cohort` are missing from the `participant.tsv` file, `NULL` will be populated for those fields in the candidate and session tables. In those cases, later manual addition is required in order to display the entry on the frontend.
```

Comment:
Agreed. I think we can go for Cohort everywhere, else it will be messy even for us.
But BIDS -> `participants.tsv` files should also be noted to change their columns for `cohort` instead of `subproject`.

Reply:
Seems ok for @laemtl too! I will apply the changes.

Reply:
@regisoc see note that I added in the PR description. Let me know if you think I may have missed something. Thanks

Reply:
LGTM!